requirements.txt

```diff
@@ -1,17 +1,18 @@
 py-solc
-web3
-eth-tester[py-evm]
+web3[tester]>=5.0.0b2
+eth-tester
+py-evm
+eth-hash
+pysha3
```

Comment:
I dont think we use that explicitly, so we should remove it.

Reply:
`pkg_resource.require` did complain about a missing sha3 module at some point. Could be that it was required by an extra dependency but not installed (because of the problem mentioned in my last commit).